actions/upload-actions.ts

```
@@ -75,3 +87,78 @@ export async function generatePdfSummary(
+  VALUES (
+      ${userid},
+      ${fileUrl},
+      ${summary},
+      ${title},
+      ${fileName},
+  );`;
+    } catch (error) {
+      console.error("Error saving pdf sumamry", error);
+      throw error;
+    }
+  }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)